### PR TITLE
feat: profile page basic markup and styles

### PR DESCRIPTION
**Pull request by Roman-Kasko-UITF**

### CODE REVIEW

File: src/pages/user-profile/user-profile.jsx [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+          <Profile.MainNavigationItem>Benefits</Profile.MainNavigationItem>
+          <Profile.MainNavigationItem>Training</Profile.MainNavigationItem>
+          <Profile.MainNavigationItem>Assets</Profile.MainNavigationItem>
+          <Profile.MainNavigationItem>More</Profile.MainNavigationItem>
+        </Profile.MainNavigation>
+      </Profile.Header>
+      <Profile.AsideNav>
+        <Profile.AsideMenuBlock>
+          <Profile.AsideMenuItem>
+            <svg height="16" viewBox="0 0 48 48" width="16" xmlns="http://www.w3.org/2000/svg">
```
Comment (AntonTitovUITF) [inline src/pages/user-profile/user-profile.jsx:43]: You can move all svg to assets/images.

**Review by AntonTitovUITF — CHANGES_REQUESTED** [review]

File: src/app.jsx [hunk 2]
```diff
@@ -1,11 +1,12 @@
 import { Route, Routes } from 'react-router-dom';
 import { images } from 'src/assets';
 import { ExampleCommonComponent } from 'src/components';
+import { UserProfile } from 'src/pages/index';
```
[the remark this answers is not in the document]
Comment (Roman-Kasko-UITF) [reply src/app.jsx:4]: +

File: src/pages/index.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import UserProfile from 'src/pages/user-profile/user-profile';
```
[the remark this answers is not in the document]
Comment (Roman-Kasko-UITF) [reply src/pages/index.js:1]: +

File: src/pages/user-profile/user-profile.jsx [hunk 4]
```diff
@@ -0,0 +1,276 @@
+import { useState } from 'react';
+
+import userPhoto from '../../assets/images/user-photo.jpg';
```
[the remark this answers is not in the document]
Comment (Roman-Kasko-UITF) [reply src/pages/user-profile/user-profile.jsx]: +

File: src/pages/user-profile/user-profile.jsx [hunk 5]
```diff
@@ -0,0 +1,276 @@
...
+import * as Profile from './style';
```
[the remark this answers is not in the document]
Comment (Roman-Kasko-UITF) [reply src/pages/user-profile/user-profile.jsx]: +

File: src/pages/user-profile/style.js [hunk 6]
```diff
@@ -0,0 +1,139 @@
+import styled from '@emotion/styled';
```
[the remark this answers is not in the document]
Comment (Roman-Kasko-UITF) [reply src/pages/user-profile/style.js:1]: +

File: src/pages/user-profile/user-profile.jsx [hunk 7]
```diff
@@ -0,0 +1,276 @@
...
+          <Profile.Avatar src={userPhoto} />
+        </Profile.AvatarContainer>
+        <Profile.TitleContainer>
+          <Profile.Title>Mary Stone</Profile.Title>
+          <Profile.TitleButtons>
+            <Profile.HeaderButton>Request a Change</Profile.HeaderButton>
+            <Profile.HeaderButton>Gear</Profile.HeaderButton>
+          </Profile.TitleButtons>
+        </Profile.TitleContainer>
+        <Profile.MainNavigation>
```
[the remark this answers is not in the document]
Comment (Roman-Kasko-UITF) [reply src/pages/user-profile/user-profile.jsx]: +

**Pull request merged**